cw-orch/src/daemon/sync/core.rs

```diff
@@ -66,6 +66,11 @@ impl Daemon {
     pub fn channel(&self) -> Channel {
         self.state().grpc_channel.clone()
     }
+
```

Comment:
Added this back because the user of daemon doesn't have access to the underlying sender